### DISCUSSION TITLE
"Start" button stopped working

### DISCUSSION THREAD

**Discussion by Kruzzzr** [yermak/AudioBookConverter#381] [Q&A] [answered]
I've been successfully using AudioBookConverter on and off for some time on my Windows 10 system and I just ran into a problem. It won't convert any longer. When I press "Start" nothing happens, not even an error message. I've tried everything I can think of. I doesn't convert any type of audio file I've tried. Does anyone have a suggestion for how I can resolve this problem?

**Comment by user2223** [comment]
Here might be a better place for this: https://github.com/yermak/AudioBookConverter/issues/379

**Comment by Kruzzzr** [comment]
Thanks

 

Where do I find the older version, 5.6.2?

 

rs-taylor says it works fine

**Reply by iourichadour** [reply]
Had the same issue on Windows 11. Installed 6.0.1 and it now works fine. BTW I also bought the program on Steam and that still download 6.0.0 that does not work

**Comment by user2223** [comment]
You can get any old version here: https://github.com/yermak/AudioBookConverter/releases

**Comment by Kruzzzr** [comment]
Thanks for the quick reply!

**Comment by yermak** [comment, marked as answer]
I will work on more transparent error reporting.

If program stopped working it might be good idea to remove its preferences. On windows they are located %USERHOME%/Application Data/AudiobookConverter-%version%. Just remove all folder. it's very hard to diagnose if they are corrupted.
If this occur again, please raise the bug and provide as much information as you can os version. What exactly worked if info, chapters and artwork loaded correctly, etc.
